iOS_SDK/OneSignalSDK/Source/OneSignal.h

```diff
@@ -210,18 +210,23 @@ typedef NS_ENUM(NSUInteger, OSNotificationDisplayType) {
 - (void)handleMessageAction:(OSInAppMessageAction * _Nonnull)action NS_SWIFT_NAME(handleMessageAction(action:));
 @end
 
-/* OneSignal Session Types */
+/* OneSignal Influence Types */
 typedef NS_ENUM(NSUInteger, Session) {
     DIRECT,
     INDIRECT,
     UNATTRIBUTED,
     DISABLED
 };
+/* OneSignal Influence Channels */
+typedef NS_ENUM(NSUInteger, OSInfluenceChannel) {
+    IN_APP_MESSAGE,
+    NOTIFICATION,
+};
 
 @interface OSOutcomeEvent : NSObject
 
 // Session enum (DIRECT, INDIRECT, UNATTRIBUTED, or DISABLED) to determine code route and request params
-@property (nonatomic) Session session;
+@property (nonatomic) Session influenceType;
```

Comment:
This public property needs to stay named as `session` as well so it can go out before the major release.